shell/app/modules/dcos/pages/cluster-dashboard/resources-summary.scss

```diff
@@ -7,3 +7,7 @@
     color: $color-dark-8;
   }
 }
+
+.quota-usage-progress .ant-progress-circle-path {
```

Comment:
Should add to antd-overwrite as a global style. `.erda-table  .ant-progress-circle-path`